docs/core/config/dict.md

```diff
@@ -234,21 +189,23 @@ map {
 You can also access multiple SQL fields. For example
 [[setting,acl_sharing_map]] can contain:
 
-```
-map {
-  pattern = shared/shared-boxes/user/$to/$from
-  table = user_shares
-  value_field = dummy
+```[dovecot.conf]
+dict_map shared/shared-boxes/user/$to/$from {
+  sql_table = user_shares
+  value dummy {
+  }
 
-  fields {
-    from_user = $from
-    to_user = $to
+  field from_user {
```

Comment:
Looks like these haven't been updated here and in some other pages. These should be:

```
value_field dummy {
}
key_field from_user {
  value = $from
}
```

Reply:
You can grep for `pattern =` at least, maybe also `value .*{`

Reply:
I think these are ldap-specific and I arbitrarily left them as they were. This should be handled by Marco in his userdb/passdb ldap ticket DOV-6457.